fixtures/events/performance_problems/file-io-on-main-thread.json

```diff
@@ -0,0 +1 @@
nted","READ_PHONE_STATE":"not_granted","WRITE_EXTERNAL_STORAGE":"not_granted"},"type":"app"},"device":{"name":"Android SDK built for x86","family":"Android","model":"Android SDK built for x86","model_id":"QSR1.190920.001","orientation":"portrait","manufacturer":"Google","brand":"google","screen_density":2.625,"screen_dpi":420,"simulator":true,"boot_time":"2022-11-21T10:57:17.182Z","timezone":"Europe/Vienna","archs":["x86"],"id":"0f38dda2-6ef7-4d9a-b7f4-6bcc80b85b2e","language":"en","locale":"en_US","screen_height_pixels":1794,"screen_width_pixels":1080,"type":"device"},"os":{"name":"Android","version":"10","build":"QSR1.190920.001","kernel_version":"4.14.112+","rooted":false,"type":"os"},"trace":{"trace_id":"b2a33f3f79fe4a7c8de3426725a045cb","span_id":"b93d2be92cd64fd5","op":"ui.action.click","status":"ok","exclusive_time":35.177709,"client_sample_rate":1.0,"hash":"1ff9a18d6a6b09a8","type":"trace"}},"culprit":"MainActivity.add_attachment","environment":"debug","grouping_config":{"enhancements":"eJybzDRxY3J-bm5-npWRgaGlroGxrpHxxEkT1-Zm5usVp-aVFFXqaWlNZAQAKGsOFg","id":"newstyle:2019-10-29"},"hashes":[],"ingest_path":[{"version":"22.11.0","public_key":"XE7QiyuNlja9PZ7I9qJlwQotzecWrUIN91BAO7Q5R38"}],"key_id":"1336851","level":"info","location":"MainActivity.add_attachment","logger":"","metadata":{"location":"MainActivity.add_attachment","title":"MainActivity.add_attachment"},"nodestore_insert":1669031864.427658,"received":1669031861.899161,"sdk":{"name":"sentry.java.android.timber","version":"6.8.0","packages":[{"name":"maven:io.sentry:sentry","version":"6.8.0"},{"name":"maven:io.sentry:sentry-android-core","version":"6.8.0"},{"name":"maven:io.sentry:sentry-android-ndk","version":"6.8.0"},{"name":"maven:io.sentry:sentry-android-timber","version":"6.8.0"}]},"span_grouping_config":{"id":"default:2022-10-27"},"spans":[{"timestamp":1669031858.806411,"start_timestamp":1669031857.712,"exclusive_time":1094.411134,"description":"1669031858711_file.txt (4.0 kB)","op":"file.write","span_id":"054ba3a374d543eb","parent_span_id":"b93d2be92cd64fd5","trace_id":"b2a33f3f79fe4a7c8de3426725a045cb","status":"ok","data":{"blocked_ui_thread":true,"call_stack":[{"function":"onClick","in_app":true,"lineno":2,"module":"io.sentry.samples.android.MainActivity$$ExternalSyntheticLambda6","native":false},{"filename":"MainActivity.java","function":"lambda$onCreate$5$io-sentry-samples-android-MainActivity","in_app":true,"lineno":93,"module":"io.sentry.samples.android.MainActivity","native":false}],"file.path":"/data/user/0/io.sentry.samples.android/files/1669031858711_file.txt","file.size":4010},"hash":"8add714f71a52ef2"}],"start_timestamp":1669031858.677,"timestamp":1669031858.806589,"title":"MainActivity.add_attachment","transaction":"MainActivity.add_attachment","transaction_info":{"source":"component"},"type":"transaction","user":{"id":"0f38dda2-6ef7-4d9a-b7f4-6bcc80b85b2e","ip_address":"86.56.213.87","geo":{"country_code":"AT","city":"Enns","region":"Austria"}},"version":"7"}
```

Comment:
Pretty-printing this is a good idea IMO. Eventually someone will want to figure out why a certain test case does or does not work, and it's good to be able to read through the file to see what's there. Also, it'll make apparent which data is not required for detector, or needs to be anonymized, or what have you